pkg/types/health.go

```diff
@@ -0,0 +1,83 @@
+// Copyright (c) 2021 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package types
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/gardener/etcd-backup-restore/pkg/wrappers"
+	flag "github.com/spf13/pflag"
+)
+
+const (
+	// DefaultEnableEtcdLeaseRenewal is a flag where a user can choose to turn off this lease renewal feature
+	DefaultEnableEtcdLeaseRenewal = false
+	// FullSnapshotLeaseName is the name for the delta snapshot lease.
+	FullSnapshotLeaseName = "full-snapshot-revisions"
+	// DeltaSnapshotLeaseName is the name for the delta snapshot lease.
+	DeltaSnapshotLeaseName = "delta-snapshot-revisions"
```

Comment:
I think these parameters need to be configurable via CLI. Snapshot lease names should not be hardcoded into backup sidecar, because there is a contract between druid and backup sidecar about which leases are used. Since druid creates the leases, it also needs to be able to communicate these names to backup sidecar. Hence, we need CLI flags for the lease names.
You can still keep these as the default values, but ideally you should be taking these from the user via CLI flags before setting to the default values.

Reply:
Updated the code to add two flags to the `server` subcommand (`full-snapshot-lease-name` and `delta-snapshot-lease-name`) and one flag to the `compact` subcommand (`full-snapshot-lease-name`)
However these parameters remain as default values